app/web3/web3Provider.ts

```diff
@@ -1,16 +1,46 @@
+import { promisify } from "bluebird";
+import ledgerWalletProvider from "ledger-wallet-provider";
+import { Store } from "redux";
 import * as Web3 from "web3";
-import { AppState } from "../actions/constants";
+import * as Web3ProviderEngine from "web3-provider-engine";
```

Comment:
could you check if yarn build works with `web3-provider-engine`? I am pretty sure that it had some problems with minifing (it's not transpiled correctly to es5). I fixed it in `ico-transparency-monitor` project so just ask me if it doesnt work